gaitutils/videos.py

```diff
@@ -46,21 +47,49 @@ def convert_videos(vidfiles, check_only=False):
         # return True if all conversion targets already exists
         return all(p.is_file() for p in convfiles.values())
 
-    # XXX: this disables Windows protection fault dialogs
-    # needed since ffmpeg2theora may crash after conversion is complete (?)
-    SEM_NOGPFAULTERRORBOX = 0x0002  # From MSDN
-    ctypes.windll.kernel32.SetErrorMode(SEM_NOGPFAULTERRORBOX)
+    if platform.system() == 'Windows':
+        # XXX: this disables Windows protection fault dialogs
+        # needed since ffmpeg2theora may crash after conversion is complete (?)
+        SEM_NOGPFAULTERRORBOX = 0x0002  # From MSDN
+        ctypes.windll.kernel32.SetErrorMode(SEM_NOGPFAULTERRORBOX)
 
     vidconv_bin = Path(cfg.general.videoconv_path)
     vidconv_opts = cfg.general.videoconv_opts
     if not (vidconv_bin.is_file() and os.access(vidconv_bin, os.X_OK)):
         raise RuntimeError(f'Invalid configured video converter: {vidconv_bin}')
-    procs = []
+
+    procs = []    
+
     for vidfile in convfiles:
-        cmd = [vidconv_bin] + vidconv_opts.split() + [vidfile]
-        # supply NO_WINDOW flag to prevent opening of consoles
-        p = subprocess.Popen(cmd, stdout=None, creationflags=0x08000000)
+        if vidconv_opts == '':
+            """ For compatibility purposes, if no video converter options are
+            specified, pass to the converter just input file name.
+            """
+            cmd = [vidconv_bin] + [vidfile]
+        else:
+            """
+            Video converter parameters given as a list of two lists.
+            e.g. [['-i', '', '-o', ''], [1, 3]] means that the 1-st and 3-rd
```

Comment:
This parameter spec is a bit clunky (IMO). How about using a string like `'-i $INPUT -o $OUTPUT'`, where the placeholders get replaced with file names?